CMSgov/mint-app
# [EASI-2219] Global header adjustments to accommodate read-only model view

# EASI-2219

## Changes and Description

- Add `Models` to Navigation Bar
- Changed routes to go home
- Removed redirect from `/` to `/models`



## How to test this change

1. Check to see if `Models` is showing up on navigation bar
2. Clicking it will send user to `/models` and empty page. (The actual page's work is [another ticket](https://jiraent.cms.gov/browse/EASI-2236))
3. Clicking and navigating home will send user to `/`



## PR Author Review Checklist



## PR Reviewer Guidelines
- It's best to pull the branch locally and test it, rather than just looking at the code online!
- Check that all code is adequately covered by tests - if it isn't feel free to suggest the addition of tests.
- Always make comments, even if it's minor, or if you don't understand why something was done.
